test/unit/warp-routes.test.ts

```diff
@@ -0,0 +1,21 @@
+import { expect } from 'chai';
+
+import { MultiProtocolProvider, WarpCore } from '@hyperlane-xyz/sdk';
+import { FileSystemRegistry } from '../../src/registry/FileSystemRegistry.js';
+
+describe('Warp Route Configs', () => {
```

Comment:
This will catch any invalid warp configs going forward. The reason for the mistake was that previously all the configs were in one big nexus bucket, meaning they could reference each other easily. Now they're broken apart by symbol and chains